src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -612,6 +612,19 @@ protected virtual void Dispose(bool disposing)
             }
         }
 
+        /// <summary>
+        /// Indicates whether the connection should be closed when the server throws an exception.
+        /// </summary>
+        /// <param name="ex">The <see cref="Exception"/> thrown from server that is potentially fatal</param>
+        /// <returns>A <see cref="bool"/> indicating if the streams should be closed.</returns>
+        /// <remarks>
+        /// This method is invoked within the context of an exception filter and simply returns false by default.
```

Comment:
I guess we shouldn't say it's called from an exception filter since that is not always the case. I see from your other change that we only call it in an exception filter in one of a few cases.